### ISSUE TITLE
Inate spells work 100% only with 1 hero. What's the point of AD then?

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Dota2-Gameplay#18451]
### Description

E.g. Let's say you randomed any other hero besides Slark and you drafted Shadow Dance (Slark's ult). Guess what? You don't get the movement speed and the regen on Shadow Dance because these are now associated with Slarks inate ability.
So you play with a mega-nerf on this spell. The examples are many.

I hope you can see how this defeats the point of playing Ability Draft, because it's like a mega nerf on all these spells, unless chosen the traditional spells of your random hero, and not even then it's working properly.

Please re-work this so that when playing AD, these inates and facets are associated with the spell itself, no matter the hero drafted.

### Example Match ID (and possibly Timestamp)

n/a

**Comment by totkeks** [comment]
Agree. AD is pretty much broken now. More than it was before already. Abilities are not granted (agha / shard), or granted at lvl 1 but require extra skill points (5 instead of the regular 4 abilities), or are useless because their main purpose was moved to innate abilities or facets. And the innates you have don't level with your AD ulti, unless you picked your heroes original ulti.
